x-pack/test_serverless/functional/test_suites/common/discover/x_pack/visualize_field.ts

```diff
@@ -48,7 +48,11 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
     beforeEach(async () => {
       await PageObjects.common.navigateToApp('discover');
+      await PageObjects.header.waitUntilLoadingHasFinished();
+      await PageObjects.discover.waitUntilSearchingHasFinished();
```

Comment:
I wonder if it's time for a 
`await PageObjects.header.waitUntilLoadingHasFinishedAndWaitUntelSearchingHasFinished();`
funcion :)